View Controllers/TimelineViewController.m

```diff
@@ -108,8 +110,13 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     cell.usernameHandleLabel.text = [@"@" stringByAppendingString: tweet.user.screenName];
     
     //timestamp/date published
-    cell.timestampLabel.text = tweet.createdAtString;
+    NSString *timestampString = tweet.createdAtString;
+    NSDateFormatter *format = [[NSDateFormatter alloc] init];
+    [format setDateFormat:@"E MMM d HH:mm:ss Z y"];
+    NSDate *newTimestamp = [format dateFromString:timestampString];
     
+    cell.timestampLabel.text = newTimestamp.shortTimeAgoSinceNow;
```

Comment:
Should you check for nil and have a default? I would check the docs to see if `dateFromString` can return nil.